src/lib/knobs/useObject/object.tsx

```diff
@@ -15,7 +15,7 @@ const Container = styled.div`
 
 interface Props {
   name: string;
-  value: number;
+  value: object;
```

Comment:
This change caused by ReactJson typings. They state that it should be an object.